src/Pyz/Yves/ShopUi/Theme/default/components/atoms/table/table.scss

```diff
@@ -1,77 +1,38 @@
 @include shop-ui-table {
+    $name: '.table';
+
     &__body,
     tbody {
         tr {
+            background-color: $setting-color-transparent !important;
+
             &:last-child {
                 border-bottom: rem(1) solid $setting-color-gray-3;
             }
 
-            &:nth-child(even) {
-                background: none;
-            }
-        }
-    }
-
-    .text-right {
-        text-align: right;
-    }
-
-    &--customer-account {
-        td,
-        th {
-            padding: rem(8);
-        }
-
-        .checkbox {
-            margin: rem(4) 0 0;
-        }
-    }
-
-    &--order,
-    &--return {
-        th,
-        td {
-            font-size: rem(14);
-
-            @include helper-breakpoint-media-min($md-min) {
-                font-size: rem(16);
+            /* stylelint-disable */
+            table tr:last-child {
+                border-bottom: none;
```

Comment:
Looks strange, please use the parent selector.

Reply:
Added modifiers to the child table.